ui/src/dashboards/reducers/n.dashboards.ts

```diff
@@ -0,0 +1,90 @@
+// Libraries
+import {combineReducers} from 'redux'
+
+// Types
+import {NDashboard, RemoteDataState, AppState} from 'src/types'
+import {
+  SET_DASHBOARDS,
+  DashboardActionTypes,
+} from 'src/dashboards/actions/n.dashboards'
+
+// TODO: port all these actions to normalized actions
+// export enum ActionTypes {
+// √  SetDashboards = 'SET_DASHBOARDS',
+//   SetDashboard = 'SET_DASHBOARD',
+//   RemoveDashboard = 'REMOVE_DASHBOARD',
+//   DeleteDashboardFailed = 'DELETE_DASHBOARD_FAILED',
+//   EditDashboard = 'EDIT_DASHBOARD',
+//   RemoveCell = 'REMOVE_CELL',
+//   AddDashboardLabels = 'ADD_DASHBOARD_LABELS',
+//   RemoveDashboardLabels = 'REMOVE_DASHBOARD_LABELS',
+// }
+
+export interface DashboardsState {
+  byID: {
+    [uuid: string]: NDashboard
+  }
+  allIDs: string[]
+  status: RemoteDataState
+}
+
+const byID = (
+  state: DashboardsState['byID'] = {},
+  action: DashboardActionTypes
+): DashboardsState['byID'] => {
+  switch (action.type) {
+    case SET_DASHBOARDS: {
+      if (!action.normalized) {
+        return state
+      }
+
+      return {...state, ...action.normalized.entities.dashboards}
```

Comment:
can we safeguard against some of these assumptions? Maybe using `get` from lodash?